fobi_custom/plugins/form_elements/fields/observational/race_observational/fobi_form_elements.py

```diff
@@ -0,0 +1,31 @@
+from fobi.base import FormFieldPlugin, form_element_plugin_registry
+from fobi_custom.plugins.form_elements.fields.intercept.forms import RACE_CHOICES
```

Comment:
Importing `RACE_CHOICES` saves time and aligns things with the corresponding Intercept question, but it has the downside of adding `Other` as a category to the question. Methodologically this seems kind of weird to me (we're already making surveyors guess the person's race, so what does `Other` mean in that context?) but I might be overthinking it. Another important consideration is that keeping  the `Other` category also makes it easier to line things up with the ACS data.

Reply:
It makes sense to me to keep consistency with the ACS, even if it's a little weird! I think `Other` functions well as a "not sure" option.